source/org/zfin/framework/presentation/tags/CreateLinkTag.java

```diff
@@ -182,7 +182,7 @@ else if (o instanceof Term) {
         else if (o instanceof TermDTO)
             link = TermDTOPresentation.getLink((TermDTO) o);
         else if (o instanceof ExpressionResult)
```

Comment:
Seems like this line should also be changed to ExpressionResult2.  Could even use the new syntax (https://blogs.oracle.com/javamagazine/post/pattern-matching-for-instanceof-in-java-14)